tests/app/va/va_profile/test_va_profile_client.py

```diff
@@ -679,7 +679,10 @@ def test_should_raise_exception_if_recipient_has_no_permissions_old_va_profile_r
             )
 
     def test_should_raise_exception_if_recipient_has_no_permissions(self, test_va_profile_client, rmock):
-        # TODO: Note that this behavior will change once we starting using default communication item permissions
+        """
+        This behavior will change once we starting using default communication item permissions.
+        """
+
```

Comment:
aren't we already using default communication item permissions? 
Changing from a TODO to a docstring does not change that if future work is necessary, a ticket number should be included for the work. 
If we are using default comm items, then does this test need to be updated? That would warrant a ticket and a TODO comment.
If we still aren't using them, then it still warrants a TODO and a ticket for that work.

Reply:
We are not already using a default.  I added a reference to #893.